examples/react/react-app/src/dojo/generated/contractComponents.ts

```diff
@@ -14,7 +14,7 @@ export function defineContractComponents(world: World) {
                 {
                     player: RecsType.BigInt,
                     remaining: RecsType.Number,
-                    last_direction: RecsType.String,
+                    last_direction: RecsType.Number,
```

Comment:
that generated file is the only one using String, and build fails because of it.
I think it is not the scope here to sync all fix all other examples, so I can either
if I revert i, build will start to fail again